api/proto/info/info.proto

```diff
@@ -348,16 +348,30 @@ message ZInfoDeviceTasks {
   // deprecated 3
 }
 
+// ZSimcardInfo describes either empty SIM slot or a slot with a SIM card inserted.
 message ZSimcardInfo {
-  // Name is a SIM card identifier. For example ICCID if available.
+  // Name is a SIM card/slot identifier.
   // Guaranteed to be unique only in the scope of the edge node.
+  // Used as a reference from ZCellularStatus.sim_cards.
   string name = 1;
   // Reference to ZCellularModuleInfo.name
   string cell_module_name = 2;
+  // International Mobile Subscriber Identity.
   string imsi = 3;
+  // Integrated Circuit Card Identifier.
   string iccid = 4;
   reserved 5;  // legacy sim card status enum
+  // The current state of the SIM card (absent, initialized, not recognized, etc.).
+  // This state is not modeled using enum because the set of possible values differs
+  // between QMI and MBIM protocols (used to control cellular modules) and there is
```

Comment:
Shouldn't we also include whether QMI or MBIM is used so that the reader of this info knows how to interpret the integer values?

Reply:
From SIM card info [there is a reference](https://github.com/milan-zededa/eve/blob/extend-cellular-apis/api/proto/info/info.proto#L357-L358) pointing to the corresponding modem and under the modem info we [publish the protocol used](https://github.com/milan-zededa/eve/blob/extend-cellular-apis/api/proto/info/info.proto#L385).

Reply:
ack